SIG-Readme-Template.md

```diff
@@ -0,0 +1,17 @@
+# <Name> Special Interest Group
+<Goals of this SIG>
+
+## Team
```

Comment:
what we found there is not really a team. You can just point to (at)open-telemetry/dotnet-approvers and (at)open-telemetry/dotnet-maintainers. Other people are coming ad-hoc.